packages/cashier/src/Components/CashierDefault/cashier-default-side-note.jsx

```diff
@@ -24,28 +24,21 @@ const CashierDefaultSideNote = ({
                     values={{ currency_code }}
                 />
             );
-        } else if (can_change_fiat_currency) {
-            return (
+        }
+
+        return (
+            !can_change_fiat_currency && (
```

Comment:
I think there is no need to add this condition here since you are already checking it thru `is_currency_banner_visible`

Reply:
agree with you, thx for noticed